pubtools/pulplib/_impl/fake/match.py

```diff
@@ -124,6 +125,21 @@ def match_in(matcher, field, obj):
     return False
 
 
+@visit(LessThanMatcher)
+def match_less(matcher, field, obj):
+    # for datetime.dateime fields, obj_value is returned as pulp_value(iso format)
```

Comment:
```suggestion
    # for datetime.datetime fields, obj_value is returned as pulp_value(iso format)
```

But actually, I don't think it is right to have the datetime conversion embedded directly next to the less-than comparison here.  Is there a way to make it work for all of the comparisons like equals, in_ and so on?

Reply:
I added https://github.com/release-engineering/pubtools-pulplib/pull/43/commits/bf154296612cc9809ad7452697eafc1df4004a50 onto your PR which I believe should fix this problem in a better way, could you please have a look?

Reply:
Thanks. This patch gets rid of that obj value vs matcher value inconsistencies.